src/main/java/io/weaviate/client6/v1/api/Config.java

```diff
@@ -42,7 +42,7 @@ public GrpcChannelOptions grpcTransportOptions() {
     return new GrpcChannelOptions(scheme, grpcHost, grpcPort, headers, tokenProvider);
   }
 
-  abstract static class Builder<SELF extends Builder<SELF>> implements ObjectBuilder<Config> {
+  public abstract static class Builder<SELF extends Builder<SELF>> implements ObjectBuilder<Config> {
```

Comment:
Config.Builder is not intended to be public, it only defines common methods for its subclasses. Making it `private` explicitly is perhaps even better, as it's not meant to be reused across the package.

Could you please update this (set to `private`) or revert the change?

Reply:
> `Builder` should be public to be usable outside this package

Just to clarify: Config.Custom is the subclass that exposes all possible configurations.

Reply:
Yes, you're right, great thinking